content/news/2024-10-31-GCB2024/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Engaging, Learning, and Sharing at the German Conference of Bioinformatics 2024 with the Galaxy Team!"
+tease: "A recap of our Galaxy team's experience at GCB 2024, featuring insightful talks, interactive poster sessions, hands-on workshops, and memorable social events!"
+date: '2024-10-30'
+authors: Engy Nasr
+subsites: [all-eu, global, us]
+tags: [GCB-2024, Bielefeld]
+main_subsite: eu
+---
+
+The [German Conference of Bioinformatics (GCB) 2024](https://gcb2024.de/) took place in Bielefeld from September 30 to October 2, bringing together leading minds in bioinformatics to discuss the latest advancements in computational biology, microbiome research, genomics, and more. Representing the Galaxy community, Engy Nasr and Momin Saim attended, contributing with two poster presentations and participating in workshops, poster sessions, and stimulating talks. Here’s a recap of our experience and the conference highlights.
+
+![Placeholder for GCB2024 Logo](gcb2024logo.png)
+
+### Day 1: Hands-on Workshops
+
+Our GCB 2024 journey began with two in-depth workshops, where we gained valuable insights and skills in specialized areas of bioinformatics. With seven workshops running in parallel, we joined the following two sessions:
+
+1. **Computational Pangenomics Workshop:** This workshop covered pangenomics fundamentals, with applications in analyzing genetic diversity across species. We explored advanced computational tools for building and analyzing pangenomic datasets, including *PanBench*, *PanGrowth*, *Plast*, *Bifrost*, and *SANS*. The hands-on exercises illuminated new ways we could use pangenomic analysis within the Galaxy framework.
+
+2. **Bioinformatic Metagenome and Metaproteome Analysis for Microbiome Insights:** Led by leading microbiome researchers, this session provided tools and workflows for understanding metagenomic and metaproteomic data. We learned methods for assessing microbial communities, a crucial area in Galaxy’s microbial data analysis tools. 
```

Comment:
Maybe you wanted to mention this instead : [Standardizing and harmonizing NGS analysis workflows](https://gcb2024.de/Workshops.html#WS7)